frontend/src/features/Train/components/DatasetStepLayout.tsx

```diff
@@ -83,11 +86,19 @@ export const UploadDatasetPanel = ({
           Upload
           <input
             type="file"
-            accept=".csv"
+            accept={acceptedTypes}
             hidden
             onChange={(e) => {
               if (e.target.files && e.target.files[0]) {
-                uploadFile({ dataSource: dataSource, file: e.target.files[0] });
+                if(e.target.files[0].type.match(acceptedTypes) == null) {
+                  toast.error('Files can only be of type ' + acceptedTypes + '.', 
+                  {position: toast.POSITION.TOP_CENTER, autoClose: 2000});
```

Comment:
I feel like we aren't being consistent with how our toasts are being shown. If this is intentional, it's fine. It'st just a bit aesthetically off

Reply:
I would recommend standardizing the Toast component so it's of a good time length. So, @AmyCGT please go to _app.ts and change the  `<ToastContainer />` to `<ToastContainer autoClose={2000} />`, and change this current code of yours to:

```ts
toast.error('Files can only be of type ' + acceptedTypes + '.');
```